src/components/home/modals/CancelLease.vue

```diff
@@ -179,6 +202,15 @@ export default {
         },
         seedPhrase() {
             return seedLib.decryptSeedPhrase(this.secretInfo.encrSeed, Vue.ls.get('pwd'))
+        },
+        getDevice() {
+            if (this.userInfo && this.userInfo.coldAddresses) {
+                var object = JSON.parse(this.userInfo.coldAddresses)
+                if (object[this.address] && object[this.address].hasOwnProperty('device')) {
```

Comment:
In fact, `this.address` is Recipient, not cold wallet address. So get device of `object[this.address]` is wrong here.